### PR TITLE
Fix #429 line break for global note

### LINKED ISSUE #429: Line Break for „Global Note“
Hi WOK,

As already discussed, it would be nice if the „Global Note“ field (which is sticky now :-)) would support line breaks because we would like to note a longer text as some guideline for the meeting. 
Even if the long text is completely visible in the email, it would be good so see the text also completely in the minutes so it is easier to update it.

Maybe after this change it would be reasonable to be able to collapse the field like the "Participants" list because it might "waste" a lot of height?

Best regards, Christoph

### PR DESCRIPTION
**Pull request by venkatr95**

### CODE REVIEW

**Comment by migerh** [comment]
The acceptance criteria from #429:

- [x] Field should be initially 2 lines in height
- [x] Pressing ENTER in this field will add a line
- [x] If user adds more lines, the hight should increase
- [ ] Line breaks should be preserved in: browser, eMail and generated PDFs, finalized meetings
- [x] TAB should allow to leave field. Latests in this case changes shall be stored to database (and be propagated to other connected users)

Only partially checked line break preservation. They are preserved after leaving the editor as well as after finalization. But there are no line breaks in the generated HTML:

image

Unable to check: Line breaks in mails and generated pdfs.

Also, while editing the textarea grows. But if you only view the global notes (e.g. after a reload) it looks like this:

image

@derwok Can you please check if this is still acceptable or if we need further changes?

File: client/templates/minutes/minutesedit.js [hunk 1]
```diff
@@ -446,6 +446,14 @@ Template.minutesedit.events({
         aMin.update({date: aDate}).catch(onError);
     },
 
+    'keyup #editGlobalNotes' (evt) {
+        evt.preventDefault();
+        console.log("fdv");
```
Comment (migerh) [inline client/templates/minutes/minutesedit.js]: Please remove this.

**Review by migerh — CHANGES_REQUESTED** [review]

**Review by migerh — APPROVED** [review]

**Comment by derwok** [comment]
Hi,
I think the feature looks already quite good! Will be very useful for users.
Three points:

1. I think we should define a maximum height for this field. Let's make it "big" enough that usual use cases need no scrolling - but small enough that if someone pastes the Wikipedia in here, that scrolling will not be endless. So, may I add one more acceptance criteria: Lets make the maximum heigth something like `700 px`.

1. My second point is - as Michael already commented - if I freshly enter a minute with a long note, The initial height is static (2 lines), but it should also be dynamic

1. If a second user (role: Invited) views the minutes (hint: use a different browser for testing!) this user will only see 2 lines height - always! With no chance to edit something, this will never change. If the point above is solved, this problem will be smaller (user might press RELOAD to get correct height). But Premium solution would be if the "Invited" users gets correct height even if the content of the note changes by moderator while invited user is passively viewing.

**Comment by derwok** [comment]
We chose the "green" one.

**Pull request closed without merging**